tests/functional/test_regions.py

```diff
@@ -465,8 +465,9 @@ def _test_single_service_region(service_name, region_name,
                                 expected_endpoint, resolver):
     bridge = ClientEndpointBridge(resolver, None, None)
     result = bridge.resolve(service_name, region_name)
-    expected = 'https://%s' % expected_endpoint
-    assert_equal(result['endpoint_url'], expected)
+    if not expected_endpoint.startswith('http'):
```

Comment:
Very minor, but we could use `urlparse` to ensure http is the scheme and not the start of the domain.

Reply:
Updated.